ext/standard/proc_open.c

```diff
@@ -1308,12 +1306,14 @@ PHP_FUNCTION(proc_open)
 	} else {
 exit_fail:
 		_php_free_envp(env);
-		if (command) {
-			efree(command);
-		}
 		RETVAL_FALSE;
 	}
 
+	/* the command_str needs to be freed if it was created by parsing an array */
```

Comment:
Can we instead addref it in the branch where it is not created dynamically and then release it here unconditionally?